lint-staged.config.js

```diff
@@ -1,7 +1,9 @@
+const eslint = "eslint --cache --fix --quiet";
 const formatFiles = "prettier --ignore-unknown --write";
 const sortPackageJson = "better-sort-package-json";
 
 export default {
+  "!(*.{js,json,md,ts,tsx,yml}|package.json)": [eslint, formatFiles],
```

Comment:
This line and the next will match for some file patterns. All patterns shall be mutually exclusive for any file to prevent race conditions.

Reply:
Reviewed and improved here [2d2a577](https://github.com/hemilabs/token-list/pull/21/commits/2d2a5773c5cdcb1e471741d93559da748f1ccb28)